src/app/doubtfire-angularjs.module.ts

```diff
@@ -305,6 +305,10 @@ DoubtfireAngularJSModule.factory(
 DoubtfireAngularJSModule.factory('CreateNewUnitModal', downgradeInjectable(CreateNewUnitModal));
 
 // directive -> component
+DoubtfireAngularJSModule.directive(
+  'projectTasksList',
```

Comment:
are you sure that this should be `projectTaskList` and not `fProjectTaskList`

Reply:
just made the change adding f- infront of my component